examples/compose-your-graphql-router/src/main.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::Result;
+use apollo_router_core::{plugin_utils, PluggableRouterServiceBuilder};
+use std::sync::Arc;
+use tower::{util::BoxService, Service, ServiceExt};
+use tracing_subscriber::EnvFilter;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let _ = tracing_subscriber::fmt::fmt()
+        .with_env_filter(EnvFilter::try_new("info").expect("could not parse log"))
+        .init();
+
+    let current_directory = std::env::current_dir()?;
+
+    let schema = Arc::new(
+        std::fs::read_to_string(
+            current_directory
+                .parent()
+                .unwrap()
+                .parent()
+                .unwrap()
```

Comment:
I guess you could have `.parent()?.parent()?` here.

Reply:
fixed in def7ea6a